cmd/thanos/sidecar.go

```diff
@@ -237,6 +247,20 @@ func (s *extLabelSet) Get() labels.Labels {
 	return s.labels
 }
 
+func (s *extLabelSet) GetPB() []storepb.Label {
+	s.mtx.Lock()
+	defer s.mtx.Unlock()
+
+	lset := make([]storepb.Label, 0, len(s.labels))
+	for _, l := range s.labels {
+		lset = append(lset, storepb.Label{
+			Name:  l.Name,
+			Value: l.Value,
+		})
+	}
+	return lset
+}
```

Comment:
Do we use `Get()` at all anymore. If not I'd suggest we rename this one to `Get()` and convert directly in `Update()`.

Reply:
We use Get inside prom Proxy.